subxt/src/error/mod.rs

```diff
@@ -19,6 +19,8 @@ pub use dispatch_error::{
 
 // Re-expose the errors we use from other crates here:
 pub use crate::metadata::Metadata;
+#[cfg(feature = "jsonrpsee")]
+pub use jsonrpsee::types::error::ErrorObject;
```

Comment:
```suggestion
crate::macros::cfg_jsonrpsee! {
    pub use jsonrpsee;
}
```

Reply:
If we wanted this, how about adding it in `subxt/src/lib.rs` to the `pub mod ext` (and then behind the cfg macro); that's how we generally expose external deps and would make sense I reckon :)